include/zephyr/drivers/regulator.h

```diff
@@ -349,6 +373,7 @@ bool regulator_is_enabled(const struct device *dev);
  *
  * @retval 0 If regulator has been successfully disabled.
  * @retval -errno Negative errno in case of failure.
+ * @retval -ENOTSUP If regulator can not be disabled.
```

Comment:
```suggestion
* @retval -ENOTSUP If regulator disablement can not be controlled.
```

just to be in line with enable?

Reply:
changed